modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -15,6 +16,19 @@ def show
 
           render json: serialized.serializable_hash
         end
+
+        def upload
+          status, error = AppealsApi::FileValidator.new(params[:document]).call
+
+          if status == :ok
+            @document = params[:document]
```

Comment:
What is the purpose of having these variables be ivars?

Reply:
draft-mode nonsense...unnecessary.